runtime/runtime/src/lib.rs

```diff
@@ -2562,8 +2558,18 @@ mod tests {
             setup_runtime(initial_balance, initial_locked, 1);
 
         let mut free_config = RuntimeConfig::free();
-        let sha256_cost =
-            ParameterCost { gas: Gas::from(1u64), compute: Compute::from(10_000_000_000_000u64) };
+        let sha256_cost = ParameterCost {
+            gas: Gas::from(1_000_000u64),
+            compute: if checked_feature!(
+                "stable",
+                ComputeCosts,
+                apply_state.current_protocol_version
+            ) {
+                Compute::from(10_000_000_000_000u64)
+            } else {
+                Compute::from(1_000_000u64)
+            },
+        };
```

Comment:
Unless you want to add a test that checks a version before the feature was added, I don't think you need this `checked_feature!`, right? (Assuming we bump the protocol version first)

Reply:
I've addressed this in https://github.com/near/nearcore/pull/8915